### ISSUE TITLE
[Bug] All Users in permissionstab are VIPs instead of Regulars

### ISSUE THREAD

**Issue by TheBlackHawk1337** [PhantomBot/PhantomBot#2654]
PhantomBot Version: 3.5.0.10
Web Browser Version (if Panel related): 1.5.1

Detailed Description:
All users are shown as VIPs.
But with chat command "!permission username" the right permission is shown (f.ex. subscriber)

Screenshots (if applicable):
![image](https://user-images.githubusercontent.com/47850314/151065182-d8c83c46-fa49-4ec5-8865-6f883de89a18.png)
![image](https://user-images.githubusercontent.com/47850314/151065452-13bc8439-ea8e-4695-879a-4bf20e248e32.png)

**Comment by Psychoboy** [comment]
This issue is more drastic, if you do the Subscriber/VIP swap subscribers become VIP then

**Comment by gmt2001** [comment]
Pretty sure this is a visual issue

**Comment by Psychoboy** [comment]
I tested this with !permission against another and they for sure were NOT vip and it returned VIP. Also functions set to VIP was letting subscribers execute even though I had it set to VIP is higher then SUB

**Comment by gmt2001** [comment]
Checked the permissions display. The OP is a visual issue due to the Permissions page on the panel not handling the Subscriber/VIP Swap flag

Psychoboy seems to be having a separate issue

**Comment by Psychoboy** [comment]
I think I found the problem, well in my case anyways. If the VIPSwap is enabled. in the DB the command is set to value of 3 instead of 5 for permission. in commandRegister it swaps them again so now the VIP is back to 5 instead of 3. then in customCommand it goes down to case 5 and it gets approved for a sub instead of being restricted it if it was VIP only

**Comment by Psychoboy** [comment]
my issue may be separate but noticed it while looking at the original reported issue

**Comment by TheBlackHawk1337** [comment]
[REOPEN]

I upgraded to Version 3.5.0.11 (stable) (Revision: b390b31). Now all  entrys showing "Subscribers" instead of "VIPs", but this is not true either. 

I filtered by "Regulars" and there are only 4 entries, all of which are ex-mods where I set the permission manually.
Filtering by "viewers" does not show any matching entries.

I tested a random "subscriber" with /user in chat and that user is not currently a subscriber. And the bot gives out that he has VIP Permissions:
![image](https://user-images.githubusercontent.com/47850314/153833936-0a098a86-ceb5-4eab-8df9-5d73f4c8ab53.png)
("Vorher 2 Monate abonniert" means Subscribed 2 months before. The user is currently not a Sub)
![image](https://user-images.githubusercontent.com/47850314/153833907-a1a51dc8-2549-42b8-a930-71daa19be76d.png)
![image](https://user-images.githubusercontent.com/47850314/153833922-c83e651f-2841-4647-802f-e79ebfb1e7dc.png)
All Screenshots are with same user.
